packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -38,9 +38,13 @@ const _removeGranuleFromCmr = async (granule, collectionId) => {
  * @param {Object} params
  * @param {Knex} params.knex - DB client
  * @param {Object} params.pgGranuleRecord - A Postgres granule record
- * @param {string} params.pgCollection - A Postgres Collection record
- * @param {Object} params.granulePgModel - Instance of granules model for PostgreSQL
- * @param {Object} params.granuleDynamoModel - Instance of granules model for DynamoDB
+ * @param {string} [params.pgCollection] - A Postgres Collection record
+ * @param {Object} [params.granulePgModel=new GranulePgModel()]
+ * - Instance of granules model for PostgreSQL
+ * @param {Object} [params.granuleDynamoModel=new models.Granule()]
+ * - Instance of granules model for DynamoDB
+ * @param {Function} [removeGranuleFromCmrFunction=_removeGranuleFromCmr()]
```

Comment:
What is the type of this function? What arguments should it take, and what should it return?

Reply:
3ab40c4345